tests/test_impl_loader.py

```diff
@@ -0,0 +1,21 @@
+import os
+import unittest
+
+from test_relative_import.relative_import_class import Sample
+from getgauge.impl_loader import update_step_resgistry_with_class
+
+
+class ImplLoaderTest(unittest.TestCase):
+    def setUp(self):
+        self.relative_file_path = os.path.join('..', 'test_relative_import', 'relative_import_class.py')
+
+    def test_update_step_resgistry_with_class(self):
+        curr_dir = os.getcwd()
+        os.chdir('tests')
+        method_list = update_step_resgistry_with_class(Sample(), self.relative_file_path)
+        os.chdir(curr_dir)
+        self.assertEqual(2, len(method_list))
```

Comment:
```suggestion
        self.assertEqual(["Greet <name> from inside the class",
            "Greet <name> from outside the class"],
        [method.step_text for method in method_list])```

Will be good to make this assertion specific